raster3d/r3.in.lidar/info.c

```diff
@@ -104,7 +105,7 @@ int scan_bounds(LASReaderH LAS_reader, int shell_style, int extents, int update,
     first = TRUE;
 
     /* init to nan in case no points are found */
-    min_x = max_x = min_y = max_y = min_z = max_z = 0.0 / 0.0;
+    min_x = max_x = min_y = max_y = min_z = max_z = INFINITY;
```

Comment:
This change is not Cmake related and should be separated out

(into a new INFINITY related PR)